language-parser/target/classes/nerdalert/input.dcf

```diff
@@ -7,4 +7,17 @@ string = "string";
 num = num + secNum;
 print(num);
 num = sqrt(28);
-print(num);
\ No newline at end of file
+print(num);
+
+if (2 == 5)
```

Comment:
I'll clean this up when I get to the parser tests.  That way we don't need the input file full of test cases as a proof of concept.